x/tokenfactory/keeper/genesis.go

```diff
@@ -9,13 +9,14 @@ import (
 // InitGenesis initializes the tokenfactory module's state from a provided genesis
 // state.
 func (k Keeper) InitGenesis(ctx sdk.Context, genState types.GenesisState) {
-	k.CreateModuleAccount(ctx)
-
 	if genState.Params.DenomCreationFee == nil {
 		genState.Params.DenomCreationFee = sdk.NewCoins()
 	}
 	k.SetParams(ctx, genState.Params)
 
+	// The call to GetModuleAccount creates a module account if it does not exist.
+	k.accountKeeper.GetModuleAccount(ctx, types.ModuleName)
+
```

Comment:
I removed the `k.CreateModuleAccount(ctx)` and replaced it with this (this is what we do in other modules, unsure why we changed it here)